install.sh

```diff
@@ -558,24 +527,16 @@ while true; do
     --cmake-arg)
         cmake_common_options+=("${2}")
         shift 2 ;;
+    --rmake_invoked)
+        rmake_invoked=true
+        shift ;;
     --) shift ; break ;;
     *)  echo "Unexpected command line parameter received; aborting";
         exit 1
         ;;
```

Comment:
Could try to guard only on rmake_invoked but the bi-directional calling support may not be something to keep longer term

Reply:
Copied rocblas for now. Would be great to have linux dependency installation somewhere else, install.sh could be very straightforward then. Would be another ticket.